legend-depot-pure-model-context/src/main/java/org/finos/legend/depot/server/resources/pure/model/context/PureModelContextResource.java

```diff
@@ -66,5 +71,25 @@ public Response getPureModelContextData(@PathParam("groupId") String groupId,
         return handle(GET_VERSION_ENTITIES_AS_PMCD, () -> service.getPureModelContextData(groupId, artifactId, versionId, clientVersion, transitive), request, () -> EtagBuilder.create().withGAV(groupId, artifactId, versionId).withProtocolVersion(clientVersion).build());
     }
 
+    @POST
+    @Path("projects/dependencies/pureModelContextData")
+    @ApiOperation(GET_VERSIONS_DEPENDENCY_ENTITIES_AS_PMCD)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getPureModelContextData(@ApiParam("projectDependencies") List<ProjectVersion> projectDependencies,
+                                            @QueryParam("clientVersion") String clientVersion,
+                                            @QueryParam("transitive") @DefaultValue("true")
+                                            @ApiParam("Whether to return transitive dependencies") boolean transitive,
+                                            @QueryParam("includeOrigin") @DefaultValue("true")
+                                                @ApiParam("Whether to return start of dependency tree") boolean includeOrigin,
+                                            @Context Request request)
+    {
+        return handle(GET_VERSIONS_DEPENDENCY_ENTITIES_AS_PMCD, () -> service.getDependenciesPureModelContextData(projectDependencies, clientVersion, transitive, includeOrigin), request, () ->
+        {
```

Comment:
will the etag still work if the project dependencies are the same but in different order?

Reply:
no, removed for now